engines/bastion_katello/app/assets/javascripts/bastion_katello/gpg-keys/new/new-gpg-key.controller.js

```diff
@@ -26,8 +29,9 @@ angular.module('Bastion.gpg-keys').controller('NewGPGKeyController',
                     $scope.table.addRow(content);
                     $scope.uploadStatus = 'success';
                     $scope.transitionTo('gpgKeys.index');
+                    GlobalNotification.setSuccessMessage(translate('GPG key %s has been created.').replace('%s', content.name));
                 } else {
-                    GlobalNotification.setErrorMessage("An error occurred while creating the GPG key: " + content.displayMessage);
+                    $scope.errorMessages = [translate("An error occurred while creating the GPG key: ") + content.displayMessage];
```

Comment:
Because to me it seems odd to put an error message in the global notification spot when the error is on the form itself.  Showing the error right above the form calls attention to the form, where the error is.